src/main/java/teammates/storage/entity/BaseComment.java

```diff
@@ -0,0 +1,21 @@
+package teammates.storage.entity;
+
+import java.util.Date;
+
+import com.google.appengine.api.datastore.Text;
+
+import teammates.common.datatransfer.CommentSendingState;
+
+public interface BaseComment {
```

Comment:
Have you tested that:
1. in the old version (the version before this change), create some comments & feedback response comments.
2. then go to this version, to see whether student or instructor still can see these comments.

just to ensure that no migration is affected